md/rest_extensions.md

```diff
@@ -19,18 +22,29 @@ Get list of running cases
 |search|string|false|Search keyword, will search in case id and name of the initiator|
 
 ## `/bici-process-api`
-### `GET`
+#### Http method
+`GET`
+
+#### Description
 Get processes
+
 #### Query parameters
 No parameters
 
 ## `/bici-query-api`
-### `GET`
+#### Http method
+`GET`
+
+#### Description
 Execute a query
+
 #### Query parameters
 |Parameter|Type|Required|Description|
 |---------|----|--------|-----------|
 |c|integer|false|Number of elements to return|
 |p|integer|false|Page to return, start at 0|
+|queryName|string|false|Name of the analytic query |
+|processName|string|true|Name of the process|
+|processVersion|string|false|Version of the process. If empy, use all allowed versions|
 |parameters|object|false|Parameters required by the query|
```

Comment:
parameters is a parameter of the query?

Reply:
it's a set of parameters, depends on the query